src/main/java/net/snowflake/client/util/SecretDetector.java

```diff
@@ -117,7 +117,9 @@ public static boolean isSensitive(String name) {
    */
   private static boolean isSensitiveParameter(String name) {
     Pattern PASSWORD_IN_NAME =
-        Pattern.compile(".*?(password|pwd|token|proxyuser).*?", Pattern.CASE_INSENSITIVE);
+        Pattern.compile(
```

Comment:
I think we should extend SecretDetectorTest adding test case for private key.

Reply:
Done